.github/workflows/website.yaml

```diff
@@ -0,0 +1,49 @@
+name: Deploy Hugo Website to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main # Adjust to your default branch name
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      # Set up Hugo
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: 'latest'
+
+      # Install Node.js (Optional, if themes require npm dependencies)
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
```

Comment:
[v22 is the active LTS](https://nodejs.org/en/about/previous-releases), previous ones are maintenance LTS so if possible, I'd switch to that.

Reply:
yep.

Reply:
done.